tutorials/2-intro-to-brian-synapses.ipynb

```diff
@@ -583,7 +607,9 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
+   "metadata": {
+    "collapsed": true
```

Comment:
Does this have any significance?

Reply:
It's a new field that recent jupyter notebook versions added, but I'm not sure about what it actually does... Maybe we should use a filter (see e.g. http://timstaley.co.uk/posts/making-git-and-jupyter-notebooks-play-nice/) to make this cleaner in the future?